modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -3815,82 +3811,6 @@ public void testLeaseReadAfterSegmentation() throws Exception {
         assertTrue(res.get().isOk());
     }
 
-    /**
-     * Tests propagation of HLC on heartbeat request and response.
-     */
-    @Test
-    public void testHlcPropagation() throws Exception {
```

Comment:
You've left lot's of dead code. E.g. there's no longer need to provide HybridClock to Loza nor store it in NodeOptions. It's just an example. I guess there's more code to remove since we no longer need. Curious why it's not a dedicated PR?

Reply:
I believe this doesn't break anything. Created a ticket for cleaning up:
https://issues.apache.org/jira/browse/IGNITE-23742

Reply:
Ticket description is empty. Anyone who is not in context won't understand what to do.

Reply:
Added the description.